arcaflow_plugin_uperf/uperf_schema.py

```diff
@@ -131,6 +124,14 @@ class ProfileFlowOpConnection(ProfileFlowOpCommon):
             "description": "The SSL Engine used by OpenSSL",
         },
     )
+    port: int = field(
+        default=20001,
+        metadata={
+            "name": "port",
+            "description": "The port used to establish the connection."
+                           " This is not the same as the comm_port.",
+        },
+    )
```

Comment:
The phrase "the connection" doesn't strike me as particularly meaningful.  This is _not_ the port used by Uperf for measuring network performance, right?  (That's "comm_port" in this lingo, right?)  It would be good if we could be more positively descriptive, here.

FWIW, the Uperf manual refers to the port specified by `-P` as the "master port".  It might be a good idea to use their terminology.

Reply:
I clarified what connection it's talking about. This is the master port, not used for performance tests themselves, but used for the communication between the master/client and slave/server. Their terminology is not very clear. We should not strive to use their documentation.

Reply:
> We should not strive to use their documentation.

I disagree:  if there are any questions, our users are going to go to _their_ documentation, such as it is.  (Even [IBM defers to them](https://www.ibm.com/docs/en/linux-on-systems?topic=recommendations-network-workload), apparently with much consternation, based on the text in their page footer.)